atd-vze/src/views/Crashes/Crash.js

```diff
@@ -134,6 +136,19 @@ function Crash(props) {
           }`}</h2>
         </Col>
       </Row>
+      {(isPrivateDrive !== false || isInAustinFullPurpose !== true) && (
+        <Row>
+          <Col>
+            <Alert color="warning">
+              This crash is not included in Vision Zero statistical reporting
+              because{" "}
+              {isPrivateDrive
+                ? "it occurred on a private drive"
+                : "it is located outside of Austin Full Purpose"}
```

Comment:
upper case or lower case Austin Full Purpose?

Reply:
let's go with: `"it is located outside of the Austin full purpose jurisdiction"`